docs/source/changelog/0.19.0-changelog.rst

```diff
@@ -0,0 +1,587 @@
+   Do not worry if you installed Manim with any previous methods, like homebrew, pip,
+   choco, or scoop. Those methods will still work, and are not deprecated. However,
+   the recommended way to install Manim is now with `uv <https://docs.astral.sh/uv/>`_.
+
+Contributors
+============
+
+A total of 54 people contributed to this
+release. People with a '+' by their names authored a patch for the first
+time.
+
+* Aarush Deshpande
+* Abulafia
+* Achille Fouilleul +
+* Benjamin Hackl
+* CJ Lee +
+* Cameron Burdgick +
+* Chin Zhe Ning
+* Christopher Hampson +
+* ChungLeeCN +
+* Eddie Ruiz +
+* F. Muenkel +
+* Francisco Manríquez Novoa
+* Geoo Chi +
+* Henrik Skov Midtiby +
+* Hugo Chargois +
+* Irvanal Haq +
+* Jay Gupta +
+* Laifsyn +
+* Larry Skuse +
+* Nemo2510 +
+* Nikhil Iyer
+* Nikhila Gurusinghe +
+* Rehmatpal Singh +
+* Romit Mohane +
+* Saveliy Yusufov +
+* Sir James Clark Maxwell
+* Sophia Wisdom +
+* Tristan Schulz
+* VPC +
+* Victorien
+* Xiuyuan (Jack) Yuan +
+* alembcke
+* anagorko +
+* czuzu +
+* fogsong233 +
+* jkjkil4 +
+* modjfy +
+* nitzanbueno +
+* yang-tsao +
+
+
+The patches included in this release have been reviewed by
+the following contributors.
+
+* Aarush Deshpande
+* Achille Fouilleul
+* Benjamin Hackl
+* Christopher Hampson
+* Eddie Ruiz
+* Francisco Manríquez Novoa
+* Henrik Skov Midtiby
+* Hugo Chargois
+* Irvanal Haq
+* Jay Gupta
+* Jérome Eertmans
+* Nemo2510
+* Nikhila Gurusinghe
+* OliverStrait
+* Saveliy Yusufov
+* Sir James Clark Maxwell
+* Tristan Schulz
+* VPC
+* Victorien
+* Xiuyuan (Jack) Yuan
+* alembcke
+* github-advanced-security[bot]
+
+Pull requests merged
+====================
+
+A total of 138 pull requests were merged for this release.
+
+Highlights
+----------
+
+* :pr:`3501`: Replaced external ``ffmpeg`` dependency with ``pyav``
+   This change removes the need to have ``ffmpeg`` available as a command line tool
+   when using Manim. While ``pyav`` technically also uses ``ffmpeg`` internally,
+   the maintainers of ``pyav`` distribute it in their binary wheels.
+
+
+* :pr:`3518`: Created a :class:`.HSV` color class, and added support for custom color spaces
+   This extends the color system of Manim and adds support to implement custom color spaces.
+   See the implementation of :class:`.HSV` for a practical example.
+
+
+* :pr:`3930`: Completely reworked the installation instructions
+   As a consequence of removing the need for the external ``ffmpeg`` dependency,
+   we have reworked and massively simplified the installation instructions. Given
+   that practically, user-written scenes are effectively small self-contained Python
+   projects, the new instructions strongly recommend using the
+   `project and dependency management tool uv <https://docs.astral.sh/uv/>`__ to ensure
+   a consistent and reproducible environment.
+
+
+* :pr:`3967`: Added support for Python 3.13
+   This adds support for Python 3.13, which brings the range of currently supported
+   Python versions to 3.9 to 3.13.
```

Comment:
```suggestion
   Python versions from 3.9 to 3.13.
```

Reply:
I think the current version is correct: "with brings the range  ... to 3.9 – 3.13."

Reply:
```suggestion
   Python versions to 3.9 -- 3.13.
```